v2.1/admin-ui-statements-page.md

```diff
@@ -110,7 +110,8 @@ The table below the statistics box provides the following details:
 Parameter | Description
 -----|------------
 App | Name of the application specified by the [`application_name`](https://www.cockroachlabs.com/docs/dev/show-vars.html#supported-variables) session setting. The **Statements Details** page shows the details for this application.
-Used DistSQL? | Indicates whether the statement (or multiple statements having the same fingerprint) were executed using DistSQL.
+Distributed execution? | Indicates whether the statement (or multiple statements having the same fingerprint) were executed using DistSQL.
```

Comment:
Do we still want to use "DistSQL" to describe this?

Reply:
Well..it is DistSQL, right?

Reply:
Yes, I just mean I thought we were moving away from using that specific phrase.  cc @knz who made the UI change.

Reply:
Huh! Didn't know that. Will wait for @knz to weigh in.

Reply:
Fixed.